variables.tf

```diff
@@ -158,3 +158,17 @@ variable "aws_route_53_validation" {
   })
   default = null
 }
+
+variable "headers" {
+  description = "Set up a custom HTTP header for fastly service "
+  type = list(object({
+    name          = string
+    type          = string
+    action        = string
+    destination   = string
+    source        = string
+    ignore_if_set = bool
+    priority      = number
+  }))
+  default = null
```

Comment:
I just realised that here we should set as default value `[]`, in the same way as we have for other list of objects, right? For example:

* https://github.com/magnolia-sre/terraform-fastly-cdn-service/pull/7/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR93
* https://github.com/magnolia-sre/terraform-fastly-cdn-service/pull/7/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR117

Reply:
True, I will update the PR with this little change.